docs/getting-started/using-testnet.md

```diff
@@ -0,0 +1,46 @@
+# Using Testnet Guide
+
+last updated for v1.0.0/testnet2
+
+---
+
+## Why Use Testnet?
+
+Testnet2 is a wonderful place where you can experiment with the Decred applications without worrying that a mistake will cost you real money. It is actually recommended that people use the Testnet2 to learn the basics of the Decred software or any new features.
```

Comment:
Might be confusing to refer to it as testnet2 here.  Probably should just say testnet.  You could have a little blurb somewhere to say that the internal name is testnet2 since we reset the network periodically and this is the 2nd iteration of it, but generally I think we should just call it testnet.